.git-crypt/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Do not edit this file.  To specify the files to encrypt, create your own
```

Comment:
Let's add whole content of `.git-crypt` folder to `.gitignore`, as well as below ones:
- .idea
- .vscode
- git-crypt-key

Reply:
Done